### PR TITLE
Inware, Bose Music, Gboard & Material Catalog icons

### PR DESCRIPTION
**Pull request by evowizz**
This PR adds the following icons:
- Inware (package: `com.evo.inware`)
- Bose Music (package: `com.bose.bosemusic`)
- Gboard (package: `com.google.android.inputmethod.latin`)
- Material Catalog (package: `io.material.catalog`)

Icons preview

### CODE REVIEW

File: app/src/main/res/drawable/bose_music.xml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
+    android:width="192dp"
+    android:height="192dp"
+    android:viewportWidth="192"
+    android:viewportHeight="192">
+  <path
+      android:pathData="M96,80L158.672,38.219C162.659,35.561 168,38.419 168,43.211V154C168,157.314 165.314,160 162,160H30C29.793,160 29.588,159.99 29.386,159.969C26.361,159.662 24,157.107 24,154V43.211C24,42.912 24.021,42.62 24.061,42.336C24.662,38.083 29.59,35.727 33.328,38.219L96,80ZM106.817,87.211L156,54.422V120L106.817,87.211ZM89.349,89.988C89.347,89.987 89.345,89.986 89.344,89.985L36,54.422V148H156V134.422L89.349,89.988Z"
+      android:fillColor="#000000"
+      android:fillType="evenOdd"/>
+</vector>
```
Comment (patrykmichalik) [inline app/src/main/res/drawable/bose_music.xml:10]:
```suggestion
<vector
    xmlns:android="http://schemas.android.com/apk/res/android"
    android:name="vector"
    android:width="192dp"
    android:height="192dp"
    android:viewportWidth="192"
    android:viewportHeight="192">
    <path
        android:name="path"
        android:pathData="M 27.196 34.695 C 29.163 33.656 31.543 33.79 33.38 35.043 L 96 77.738 L 158.62 35.043 C 160.458 33.79 162.837 33.656 164.804 34.695 C 166.77 35.734 168 37.776 168 40 L 168 152 C 168 155.314 165.314 158 162 158 L 30 158 C 26.686 158 24 155.314 24 152 L 24 40 C 24 37.776 25.23 35.734 27.196 34.695 Z M 106.651 85 L 156 118.647 L 156 51.353 L 106.651 85 Z M 156 133.171 L 92.62 89.957 L 36 51.353 L 36 146 L 156 146 L 156 133.171 Z"
        android:fillColor="#000000"
        android:strokeWidth="1"
        android:fillType="evenOdd"/>
</vector>
```

File: app/src/main/res/drawable/gboard.xml [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+      android:fillColor="#000000"/>
+  <path
+      android:pathData="M142.545,133.818C142.545,132.212 143.848,130.909 145.455,130.909H157.091C158.698,130.909 160,132.212 160,133.818V145.455C160,147.061 158.698,148.364 157.091,148.364H145.455C143.848,148.364 142.545,147.061 142.545,145.455V133.818Z"
+      android:fillColor="#000000"/>
+  <path
+      android:pathData="M96,48.727C86.159,48.727 78.182,56.705 78.182,66.546C78.182,76.386 86.159,84.364 96,84.364C104.465,84.364 111.552,78.46 113.367,70.546H96C93.791,70.546 92,68.755 92,66.546C92,64.336 93.791,62.546 96,62.546H117.818C120.027,62.546 121.818,64.336 121.818,66.546C121.818,80.804 110.259,92.364 96,92.364C81.741,92.364 70.182,80.804 70.182,66.546C70.182,52.286 81.741,40.727 96,40.727C103.128,40.727 109.589,43.621 114.256,48.289C115.818,49.851 115.818,52.384 114.256,53.946C112.694,55.508 110.161,55.508 108.599,53.946C105.371,50.717 100.922,48.727 96,48.727Z"
+      android:fillColor="#000000"
+      android:fillType="evenOdd"/>
+</vector>
```
Comment (patrykmichalik) [inline app/src/main/res/drawable/gboard.xml:29]:
```suggestion
<vector
    xmlns:android="http://schemas.android.com/apk/res/android"
    android:name="vector"
    android:width="192dp"
    android:height="192dp"
    android:viewportWidth="192"
    android:viewportHeight="192">
    <path
        android:name="path_1"
        android:pathData="M 64 28 C 62.895 28 62 28.895 62 30 L 62 79.542 C 62 80.235 62.358 80.879 62.948 81.243 L 96 101.694 L 129.052 81.243 C 129.641 80.879 130 80.235 130 79.542 L 130 30 C 130 28.895 129.105 28 128 28 L 64 28 Z M 50 30 C 50 22.268 56.268 16 64 16 L 128 16 C 135.732 16 142 22.268 142 30 L 142 79.542 C 142 84.392 139.49 88.896 135.366 91.448 L 101.262 112.55 C 98.037 114.545 93.962 114.545 90.738 112.55 L 56.634 91.448 C 52.51 88.896 50 84.392 50 79.542 L 50 30 Z M 32 90 C 29.791 90 28 91.791 28 94 L 28 162 C 28 164.209 29.791 166 32 166 L 160 166 C 162.209 166 164 164.209 164 162 L 164 94 C 164 91.791 162.209 90 160 90 L 156 90 C 152.686 90 150 87.314 150 84 C 150 80.686 152.686 78 156 78 L 160 78 C 168.837 78 176 85.163 176 94 L 176 162 C 176 170.837 168.837 178 160 178 L 32 178 C 23.163 178 16 170.837 16 162 L 16 94 C 16 85.163 23.163 78 32 78 L 36 78 C 39.314 78 42 80.686 42 84 C 42 87.314 39.314 90 36 90 L 32 90 Z M 81 59 C 81 50.716 87.716 44 96 44 C 100.143 44 103.888 45.675 106.607 48.393 C 108.169 49.956 110.701 49.956 112.263 48.393 C 113.826 46.831 113.826 44.299 112.263 42.736 C 108.106 38.579 102.35 36 96 36 C 83.297 36 73 46.297 73 59 C 73 71.703 83.297 82 96 82 C 108.703 82 119 71.703 119 59 C 119 56.791 117.209 55 115 55 L 96 55 C 93.791 55 92 56.791 92 59 C 92 61.209 93.791 63 96 63 L 110.461 63 C 108.71 69.342 102.899 74 96 74 C 87.716 74 81 67.284 81 59 Z M 56 152 C 56 148.686 58.686 146 62 146 L 130 146 C 133.314 146 136 148.686 136 152 C 136 155.314 133.314 158 130 158 L 62 158 C 58.686 158 56 155.314 56 152 Z M 48 138 C 52.418 138 56 134.418 56 130 C 56 125.582 52.418 122 48 122 C 43.582 122 40 125.582 40 130 C 40 134.418 43.582 138 48 138 Z M 56 106 C 56 110.418 52.418 114 48 114 C 43.582 114 40 110.418 40 106 C 40 101.582 43.582 98 48 98 C 52.418 98 56 101.582 56 106 Z M 72 138 C 76.418 138 80 134.418 80 130 C 80 125.582 76.418 122 72 122 C 67.582 122 64 125.582 64 130 C 64 134.418 67.582 138 72 138 Z M 128 130 C 128 134.418 124.418 138 120 138 C 115.582 138 112 134.418 112 130 C 112 125.582 115.582 122 120 122 C 124.418 122 128 125.582 128 130 Z M 144 138 C 148.418 138 152 134.418 152 130 C 152 125.582 148.418 122 144 122 C 139.582 122 136 125.582 136 130 C 136 134.418 139.582 138 144 138 Z M 152 106 C 152 110.418 148.418 114 144 114 C 139.582 114 136 110.418 136 106 C 136 101.582 139.582 98 144 98 C 148.418 98 152 101.582 152 106 Z M 96 138 C 100.418 138 104 134.418 104 130 C 104 125.582 100.418 122 96 122 C 91.582 122 88 125.582 88 130 C 88 134.418 91.582 138 96 138 Z"
        android:fillColor="#000000"
        android:strokeWidth="1"
        android:fillType="evenOdd"/>
</vector>
```

File: app/src/main/res/drawable/inware.xml [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+      android:fillType="evenOdd"/>
+  <path
+      android:pathData="M83.4,18.3C83.4,14.821 80.579,12 77.1,12C73.621,12 70.8,14.821 70.8,18.3V37.2H66.6C50.363,37.2 37.2,50.363 37.2,66.6V70.8H18.3C14.821,70.8 12,73.621 12,77.1C12,80.579 14.821,83.4 18.3,83.4H37.2V108.6H18.3C14.821,108.6 12,111.421 12,114.9C12,118.379 14.821,121.2 18.3,121.2H37.2V125.4C37.2,141.637 50.363,154.8 66.6,154.8H70.8V173.7C70.8,177.179 73.621,180 77.1,180C80.579,180 83.4,177.179 83.4,173.7V154.8H108.6V173.7C108.6,177.179 111.421,180 114.9,180C118.379,180 121.2,177.179 121.2,173.7V154.8H125.4C141.637,154.8 154.8,141.637 154.8,125.4V121.2H173.7C177.179,121.2 180,118.379 180,114.9C180,111.421 177.179,108.6 173.7,108.6H154.8V83.4H173.7C177.179,83.4 180,80.579 180,77.1C180,73.621 177.179,70.8 173.7,70.8H154.8V66.6C154.8,50.363 141.637,37.2 125.4,37.2H121.2V18.3C121.2,14.821 118.379,12 114.9,12C111.421,12 108.6,14.821 108.6,18.3V37.2H83.4V18.3ZM66.6,49.8H125.4C134.678,49.8 142.2,57.322 142.2,66.6V125.4C142.2,134.678 134.678,142.2 125.4,142.2H66.6C57.322,142.2 49.8,134.678 49.8,125.4V66.6C49.8,57.322 57.322,49.8 66.6,49.8Z"
+      android:fillColor="#000000"
+      android:fillType="evenOdd"/>
+</vector>
```
Comment (patrykmichalik) [inline app/src/main/res/drawable/inware.xml:14]:
```suggestion
<vector
    xmlns:android="http://schemas.android.com/apk/res/android"
    android:name="vector"
    android:width="192dp"
    android:height="192dp"
    android:viewportWidth="192"
    android:viewportHeight="192">
    <path
        android:name="path_1"
        android:pathData="M 64 28 C 62.895 28 62 28.895 62 30 L 62 79.542 C 62 80.235 62.358 80.879 62.948 81.243 L 96 101.694 L 129.052 81.243 C 129.641 80.879 130 80.235 130 79.542 L 130 30 C 130 28.895 129.105 28 128 28 L 64 28 Z M 50 30 C 50 22.268 56.268 16 64 16 L 128 16 C 135.732 16 142 22.268 142 30 L 142 79.542 C 142 84.392 139.49 88.896 135.366 91.448 L 101.262 112.55 C 98.037 114.545 93.962 114.545 90.738 112.55 L 56.634 91.448 C 52.51 88.896 50 84.392 50 79.542 L 50 30 Z M 32 90 C 29.791 90 28 91.791 28 94 L 28 162 C 28 164.209 29.791 166 32 166 L 160 166 C 162.209 166 164 164.209 164 162 L 164 94 C 164 91.791 162.209 90 160 90 L 156 90 C 152.686 90 150 87.314 150 84 C 150 80.686 152.686 78 156 78 L 160 78 C 168.837 78 176 85.163 176 94 L 176 162 C 176 170.837 168.837 178 160 178 L 32 178 C 23.163 178 16 170.837 16 162 L 16 94 C 16 85.163 23.163 78 32 78 L 36 78 C 39.314 78 42 80.686 42 84 C 42 87.314 39.314 90 36 90 L 32 90 Z M 81 59 C 81 50.716 87.716 44 96 44 C 100.143 44 103.888 45.675 106.607 48.393 C 108.169 49.956 110.701 49.956 112.263 48.393 C 113.826 46.831 113.826 44.299 112.263 42.736 C 108.106 38.579 102.35 36 96 36 C 83.297 36 73 46.297 73 59 C 73 71.703 83.297 82 96 82 C 108.703 82 119 71.703 119 59 C 119 56.791 117.209 55 115 55 L 96 55 C 93.791 55 92 56.791 92 59 C 92 61.209 93.791 63 96 63 L 110.461 63 C 108.71 69.342 102.899 74 96 74 C 87.716 74 81 67.284 81 59 Z M 56 152 C 56 148.686 58.686 146 62 146 L 130 146 C 133.314 146 136 148.686 136 152 C 136 155.314 133.314 158 130 158 L 62 158 C 58.686 158 56 155.314 56 152 Z M 48 138 C 52.418 138 56 134.418 56 130 C 56 125.582 52.418 122 48 122 C 43.582 122 40 125.582 40 130 C 40 134.418 43.582 138 48 138 Z M 56 106 C 56 110.418 52.418 114 48 114 C 43.582 114 40 110.418 40 106 C 40 101.582 43.582 98 48 98 C 52.418 98 56 101.582 56 106 Z M 72 138 C 76.418 138 80 134.418 80 130 C 80 125.582 76.418 122 72 122 C 67.582 122 64 125.582 64 130 C 64 134.418 67.582 138 72 138 Z M 128 130 C 128 134.418 124.418 138 120 138 C 115.582 138 112 134.418 112 130 C 112 125.582 115.582 122 120 122 C 124.418 122 128 125.582 128 130 Z M 144 138 C 148.418 138 152 134.418 152 130 C 152 125.582 148.418 122 144 122 C 139.582 122 136 125.582 136 130 C 136 134.418 139.582 138 144 138 Z M 152 106 C 152 110.418 148.418 114 144 114 C 139.582 114 136 110.418 136 106 C 136 101.582 139.582 98 144 98 C 148.418 98 152 101.582 152 106 Z M 96 138 C 100.418 138 104 134.418 104 130 C 104 125.582 100.418 122 96 122 C 91.582 122 88 125.582 88 130 C 88 134.418 91.582 138 96 138 Z"
        android:fillColor="#000000"
        android:strokeWidth="1"
        android:fillType="evenOdd"/>
</vector>
```

File: app/src/main/res/drawable/material_catalog.xml [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+      android:pathData="M122,164C145.196,164 164,145.196 164,122C164,98.804 145.196,80 122,80C98.804,80 80,98.804 80,122C80,145.196 98.804,164 122,164ZM122,176C151.823,176 176,151.823 176,122C176,92.177 151.823,68 122,68C92.177,68 68,92.177 68,122C68,151.823 92.177,176 122,176Z"
+      android:fillColor="#000000"
+      android:fillType="evenOdd"/>
+  <path
+      android:pathData="M29,29H111V60.973C114.57,60.334 118.246,60 122,60C122.334,60 122.667,60.003 123,60.008V29C123,22.373 117.627,17 111,17H29C22.373,17 17,22.373 17,29V111C17,117.627 22.373,123 29,123H60.008C60.003,122.667 60,122.334 60,122C60,118.246 60.334,114.57 60.973,111H29L29,29Z"
+      android:fillColor="#000000"/>
+  <path
+      android:pathData="M89.819,111C88.64,114.45 88,118.15 88,122C88,122.334 88.005,122.668 88.014,123H111C117.627,123 123,117.627 123,111V88.014C122.668,88.005 122.334,88 122,88C118.15,88 114.45,88.64 111,89.819V111H89.819Z"
+      android:fillColor="#000000"/>
+</vector>
```
Comment (patrykmichalik) [inline app/src/main/res/drawable/material_catalog.xml:16]:
```suggestion
<vector
    xmlns:android="http://schemas.android.com/apk/res/android"
    android:name="vector"
    android:width="192dp"
    android:height="192dp"
    android:viewportWidth="192"
    android:viewportHeight="192">
    <path
        android:name="path"
        android:pathData="M 16 30 C 16 22.268 22.268 16 30 16 L 106 16 C 113.732 16 120 22.268 120 30 L 120 52.03 C 119.336 52.01 118.669 52 118 52 C 114.601 52 111.261 52.257 108 52.753 L 108 30 C 108 28.895 107.105 28 106 28 L 30 28 C 28.895 28 28 28.895 28 30 L 28 106 C 28 107.105 28.895 108 30 108 L 52.753 108 C 52.257 111.261 52 114.601 52 118 C 52 118.669 52.01 119.336 52.03 120 L 30 120 C 22.268 120 16 113.732 16 106 L 16 30 Z M 118 80 C 118.671 80 119.338 80.017 120 80.052 L 120 106 C 120 113.732 113.732 120 106 120 L 80.052 120 C 80.017 119.338 80 118.671 80 118 C 80 114.539 80.463 111.186 81.329 108 L 106 108 C 107.105 108 108 107.105 108 106 L 108 81.329 C 111.186 80.463 114.539 80 118 80 Z M 72 118 C 72 92.595 92.595 72 118 72 C 143.405 72 164 92.595 164 118 C 164 143.405 143.405 164 118 164 C 92.595 164 72 143.405 72 118 Z M 118 60 C 85.968 60 60 85.968 60 118 C 60 150.033 85.968 176 118 176 C 150.033 176 176 150.033 176 118 C 176 85.968 150.033 60 118 60 Z"
        android:fillColor="#000000"
        android:strokeWidth="1"
        android:fillType="evenOdd"/>
</vector>
```

File: svgs/bose_music.svg [hunk 5]
```diff
@@ -0,0 +1,3 @@
+<svg width="192" height="192" viewBox="0 0 192 192" fill="none" xmlns="http://www.w3.org/2000/svg">
+<path fill-rule="evenodd" clip-rule="evenodd" d="M96 80L158.672 38.2188C162.659 35.5606 168 38.4189 168 43.2111V154C168 157.314 165.314 160 162 160H30C29.7929 160 29.5882 159.99 29.3865 159.969C26.361 159.662 24 157.107 24 154V43.2111C24 42.9116 24.0209 42.6196 24.061 42.3361C24.6623 38.0832 29.5901 35.7267 33.3282 38.2188L96 80ZM106.817 87.2111L156 54.4222V120L106.817 87.2111ZM89.349 89.9882C89.3472 89.987 89.3454 89.9858 89.3436 89.9846L36 54.4222V148H156V134.422L89.349 89.9882Z" fill="black"/>
+</svg>
```
Comment (patrykmichalik) [inline svgs/bose_music.svg:3]:
```suggestion
<svg width="192" height="192" viewBox="0 0 192 192" fill="none" xmlns="http://www.w3.org/2000/svg">
<path fill-rule="evenodd" clip-rule="evenodd" d="M27.1964 34.6953C29.1627 33.6561 31.5425 33.7898 33.38 35.0426L96 77.7381L158.62 35.0426C160.458 33.7898 162.837 33.6561 164.804 34.6953C166.77 35.7345 168 37.776 168 40V130V152C168 155.314 165.314 158 162 158H30C26.6863 158 24 155.314 24 152V40C24 37.776 25.2302 35.7345 27.1964 34.6953ZM106.651 85L156 118.647V51.3528L106.651 85ZM156 133.171L92.62 89.9574L36 51.3528V146H156V133.171Z" fill="black"/>
</svg>
```

File: svgs/gboard.svg [hunk 6]
```diff
@@ -0,0 +1,9 @@
...
+<path d="M142.545 133.818C142.545 132.212 143.848 130.909 145.455 130.909H157.091C158.698 130.909 160 132.212 160 133.818V145.455C160 147.061 158.698 148.364 157.091 148.364H145.455C143.848 148.364 142.545 147.061 142.545 145.455V133.818Z" fill="black"/>
+<path fill-rule="evenodd" clip-rule="evenodd" d="M96 48.7273C86.1593 48.7273 78.1818 56.7047 78.1818 66.5455C78.1818 76.3861 86.1593 84.3636 96 84.3636C104.465 84.3636 111.552 78.4601 113.367 70.5455H96C93.7909 70.5455 92 68.7546 92 66.5455C92 64.3363 93.7909 62.5455 96 62.5455H117.818C120.027 62.5455 121.818 64.3363 121.818 66.5455C121.818 80.8044 110.259 92.3636 96 92.3636C81.741 92.3636 70.1818 80.8044 70.1818 66.5455C70.1818 52.2865 81.741 40.7273 96 40.7273C103.128 40.7273 109.589 43.6214 114.256 48.2893C115.818 49.8514 115.818 52.3841 114.256 53.9461C112.694 55.5082 110.161 55.5082 108.599 53.9461C105.371 50.7173 100.922 48.7273 96 48.7273Z" fill="black"/>
+</svg>
```
Comment (patrykmichalik) [inline svgs/gboard.svg:9]:
```suggestion
<svg width="192" height="192" viewBox="0 0 192 192" fill="none" xmlns="http://www.w3.org/2000/svg">
<path fill-rule="evenodd" clip-rule="evenodd" d="M64 28C62.8954 28 62 28.8954 62 30V79.5424C62 80.2352 62.3585 80.8787 62.9477 81.2432L96 101.694L129.052 81.2432C129.641 80.8787 130 80.2352 130 79.5424V30C130 28.8954 129.105 28 128 28H64ZM50 30C50 22.268 56.268 16 64 16H128C135.732 16 142 22.268 142 30V79.5424C142 84.3919 139.49 88.8961 135.366 91.4477L101.262 112.55C98.0375 114.545 93.9624 114.545 90.7383 112.55L56.6336 91.4477C52.5097 88.896 50 84.3919 50 79.5424V30ZM32 90C29.7909 90 28 91.7909 28 94V162C28 164.209 29.7909 166 32 166H160C162.209 166 164 164.209 164 162V94C164 91.7909 162.209 90 160 90H156C152.686 90 150 87.3137 150 84C150 80.6863 152.686 78 156 78H160C168.837 78 176 85.1634 176 94V162C176 170.837 168.837 178 160 178H32C23.1635 178 16 170.837 16 162V94C16 85.1634 23.1634 78 32 78H36C39.3137 78 42 80.6863 42 84C42 87.3137 39.3137 90 36 90H32ZM81 59C81 50.7157 87.7157 44 96 44C100.143 44 103.888 45.6746 106.607 48.3934C108.169 49.9555 110.701 49.9555 112.263 48.3934C113.826 46.8313 113.826 44.2986 112.263 42.7365C108.106 38.5787 102.35 36 96 36C83.2975 36 73 46.2975 73 59C73 71.7025 83.2975 82 96 82C108.703 82 119 71.7025 119 59C119 56.7909 117.209 55 115 55H96C93.7909 55 92 56.7909 92 59C92 61.2091 93.7909 63 96 63H110.461C108.71 69.3424 102.899 74 96 74C87.7157 74 81 67.2843 81 59ZM56 152C56 148.686 58.6863 146 62 146H130C133.314 146 136 148.686 136 152C136 155.314 133.314 158 130 158H62C58.6863 158 56 155.314 56 152ZM48 138C52.4183 138 56 134.418 56 130C56 125.582 52.4183 122 48 122C43.5817 122 40 125.582 40 130C40 134.418 43.5817 138 48 138ZM56 106C56 110.418 52.4183 114 48 114C43.5817 114 40 110.418 40 106C40 101.582 43.5817 98 48 98C52.4183 98 56 101.582 56 106ZM72 138C76.4183 138 80 134.418 80 130C80 125.582 76.4183 122 72 122C67.5817 122 64 125.582 64 130C64 134.418 67.5817 138 72 138ZM128 130C128 134.418 124.418 138 120 138C115.582 138 112 134.418 112 130C112 125.582 115.582 122 120 122C124.418 122 128 125.582 128 130ZM144 138C148.418 138 152 134.418 152 130C152 125.582 148.418 122 144 122C139.582 122 136 125.582 136 130C136 134.418 139.582 138 144 138ZM152 106C152 110.418 148.418 114 144 114C139.582 114 136 110.418 136 106C136 101.582 139.582 98 144 98C148.418 98 152 101.582 152 106ZM96 138C100.418 138 104 134.418 104 130C104 125.582 100.418 122 96 122C91.5817 122 88 125.582 88 130C88 134.418 91.5817 138 96 138Z" fill="black"/>
</svg>
```

File: svgs/inware.svg [hunk 7]
```diff
@@ -0,0 +1,4 @@
...
+<path fill-rule="evenodd" clip-rule="evenodd" d="M83.4 18.3C83.4 14.8206 80.5794 12 77.1 12C73.6206 12 70.8 14.8206 70.8 18.3V37.2H66.6C50.3628 37.2 37.2 50.3628 37.2 66.6V70.8H18.3C14.8206 70.8 12 73.6206 12 77.1C12 80.5794 14.8206 83.4 18.3 83.4H37.2V108.6H18.3C14.8206 108.6 12 111.421 12 114.9C12 118.379 14.8206 121.2 18.3 121.2H37.2V125.4C37.2 141.637 50.3628 154.8 66.6 154.8H70.8V173.7C70.8 177.179 73.6206 180 77.1 180C80.5794 180 83.4 177.179 83.4 173.7V154.8H108.6V173.7C108.6 177.179 111.421 180 114.9 180C118.379 180 121.2 177.179 121.2 173.7V154.8H125.4C141.637 154.8 154.8 141.637 154.8 125.4V121.2H173.7C177.179 121.2 180 118.379 180 114.9C180 111.421 177.179 108.6 173.7 108.6H154.8V83.4H173.7C177.179 83.4 180 80.5794 180 77.1C180 73.6206 177.179 70.8 173.7 70.8H154.8V66.6C154.8 50.3628 141.637 37.2 125.4 37.2H121.2V18.3C121.2 14.8206 118.379 12 114.9 12C111.421 12 108.6 14.8206 108.6 18.3V37.2H83.4V18.3ZM66.6 49.8H125.4C134.678 49.8 142.2 57.3216 142.2 66.6V125.4C142.2 134.678 134.678 142.2 125.4 142.2H66.6C57.3216 142.2 49.8 134.678 49.8 125.4V66.6C49.8 57.3216 57.3216 49.8 66.6 49.8Z" fill="black"/>
+</svg>
```
Comment (patrykmichalik) [inline svgs/inware.svg:4]:
```suggestion
<svg width="192" height="192" viewBox="0 0 192 192" fill="none" xmlns="http://www.w3.org/2000/svg">
<path fill-rule="evenodd" clip-rule="evenodd" d="M64 28C62.8954 28 62 28.8954 62 30V79.5424C62 80.2352 62.3585 80.8787 62.9477 81.2432L96 101.694L129.052 81.2432C129.641 80.8787 130 80.2352 130 79.5424V30C130 28.8954 129.105 28 128 28H64ZM50 30C50 22.268 56.268 16 64 16H128C135.732 16 142 22.268 142 30V79.5424C142 84.3919 139.49 88.8961 135.366 91.4477L101.262 112.55C98.0375 114.545 93.9624 114.545 90.7383 112.55L56.6336 91.4477C52.5097 88.896 50 84.3919 50 79.5424V30ZM32 90C29.7909 90 28 91.7909 28 94V162C28 164.209 29.7909 166 32 166H160C162.209 166 164 164.209 164 162V94C164 91.7909 162.209 90 160 90H156C152.686 90 150 87.3137 150 84C150 80.6863 152.686 78 156 78H160C168.837 78 176 85.1634 176 94V162C176 170.837 168.837 178 160 178H32C23.1635 178 16 170.837 16 162V94C16 85.1634 23.1634 78 32 78H36C39.3137 78 42 80.6863 42 84C42 87.3137 39.3137 90 36 90H32ZM81 59C81 50.7157 87.7157 44 96 44C100.143 44 103.888 45.6746 106.607 48.3934C108.169 49.9555 110.701 49.9555 112.263 48.3934C113.826 46.8313 113.826 44.2986 112.263 42.7365C108.106 38.5787 102.35 36 96 36C83.2975 36 73 46.2975 73 59C73 71.7025 83.2975 82 96 82C108.703 82 119 71.7025 119 59C119 56.7909 117.209 55 115 55H96C93.7909 55 92 56.7909 92 59C92 61.2091 93.7909 63 96 63H110.461C108.71 69.3424 102.899 74 96 74C87.7157 74 81 67.2843 81 59ZM56 152C56 148.686 58.6863 146 62 146H130C133.314 146 136 148.686 136 152C136 155.314 133.314 158 130 158H62C58.6863 158 56 155.314 56 152ZM48 138C52.4183 138 56 134.418 56 130C56 125.582 52.4183 122 48 122C43.5817 122 40 125.582 40 130C40 134.418 43.5817 138 48 138ZM56 106C56 110.418 52.4183 114 48 114C43.5817 114 40 110.418 40 106C40 101.582 43.5817 98 48 98C52.4183 98 56 101.582 56 106ZM72 138C76.4183 138 80 134.418 80 130C80 125.582 76.4183 122 72 122C67.5817 122 64 125.582 64 130C64 134.418 67.5817 138 72 138ZM128 130C128 134.418 124.418 138 120 138C115.582 138 112 134.418 112 130C112 125.582 115.582 122 120 122C124.418 122 128 125.582 128 130ZM144 138C148.418 138 152 134.418 152 130C152 125.582 148.418 122 144 122C139.582 122 136 125.582 136 130C136 134.418 139.582 138 144 138ZM152 106C152 110.418 148.418 114 144 114C139.582 114 136 110.418 136 106C136 101.582 139.582 98 144 98C148.418 98 152 101.582 152 106ZM96 138C100.418 138 104 134.418 104 130C104 125.582 100.418 122 96 122C91.5817 122 88 125.582 88 130C88 134.418 91.5817 138 96 138Z" fill="black"/>
</svg>
```

File: svgs/material_catalog.svg [hunk 8]
```diff
@@ -0,0 +1,5 @@
+<svg width="192" height="192" viewBox="0 0 192 192" fill="none" xmlns="http://www.w3.org/2000/svg">
+<path fill-rule="evenodd" clip-rule="evenodd" d="M122 164C145.196 164 164 145.196 164 122C164 98.804 145.196 80 122 80C98.804 80 80 98.804 80 122C80 145.196 98.804 164 122 164ZM122 176C151.823 176 176 151.823 176 122C176 92.1766 151.823 68 122 68C92.1766 68 68 92.1766 68 122C68 151.823 92.1766 176 122 176Z" fill="black"/>
+<path d="M29 29H111V60.9728C114.57 60.3336 118.246 60 122 60C122.334 60 122.667 60.0026 123 60.0079V29C123 22.3726 117.627 17 111 17H29C22.3726 17 17 22.3726 17 29V111C17 117.627 22.3726 123 29 123H60.0079C60.0026 122.667 60 122.334 60 122C60 118.246 60.3336 114.57 60.9728 111H29L29 29Z" fill="black"/>
+<path d="M89.8188 111C88.6398 114.45 88 118.15 88 122C88 122.334 88.0048 122.668 88.0144 123H111C117.627 123 123 117.627 123 111V88.0144C122.668 88.0048 122.334 88 122 88C118.15 88 114.45 88.6398 111 89.8188V111H89.8188Z" fill="black"/>
+</svg>
```
Comment (patrykmichalik) [inline svgs/material_catalog.svg:5]:
```suggestion
<svg width="192" height="192" viewBox="0 0 192 192" fill="none" xmlns="http://www.w3.org/2000/svg">
<path fill-rule="evenodd" clip-rule="evenodd" d="M16 30C16 22.268 22.268 16 30 16H106C113.732 16 120 22.268 120 30V52.0297C119.336 52.01 118.669 52 118 52C114.601 52 111.261 52.257 108 52.7527V30C108 28.8954 107.105 28 106 28H30C28.8954 28 28 28.8954 28 30V106C28 107.105 28.8954 108 30 108H52.7527C52.257 111.261 52 114.601 52 118C52 118.669 52.01 119.336 52.0297 120H30C22.268 120 16 113.732 16 106V30ZM118 80C118.671 80 119.338 80.0174 120 80.0517V106C120 113.732 113.732 120 106 120H80.0517C80.0174 119.338 80 118.671 80 118C80 114.539 80.4627 111.186 81.3295 108H106C107.105 108 108 107.105 108 106V81.3295C111.186 80.4627 114.539 80 118 80ZM72 118C72 92.5949 92.5949 72 118 72C143.405 72 164 92.5949 164 118C164 143.405 143.405 164 118 164C92.5949 164 72 143.405 72 118ZM118 60C85.9675 60 60 85.9675 60 118C60 150.033 85.9675 176 118 176C150.033 176 176 150.033 176 118C176 85.9675 150.033 60 118 60Z" fill="black"/>
</svg>
```

**Review by patrykmichalik — CHANGES_REQUESTED** [review]
Hello, and thanks for your interest in contributing! I’d suggest these changes.

**Comment by evowizz** [comment]
Thank you!

I accepted the Gboard icon as it is a different icon which is better than the one I imagined. As for the other icons, I applied your changes to my own icons, since I had the originals and it allowed me to keep them updated too.

**Review by patrykmichalik — APPROVED** [review]

**Comment by patrykmichalik** [comment]
Thanks for applying the suggested changes! I’m glad you like the Gboard icon. Cheers for the contribution!

**Pull request merged**